base_classes/NXprocess.nxdl.xml

```diff
@@ -45,12 +50,35 @@
     </field>
     <group type="NXnote">
         <doc>
-            The note will contain information about how the data was processed
-            or anything about the data provenance. 
-            The contents of the note can be anything that the processing code 
-            can understand, or simple text.
+            The note will contain information about how the data was
+            processed or anything about the data provenance. The
+            contents of the note can be anything that the processing
+            code can understand, or simple text.
             
             The name will be numbered to allow for ordering of steps.
         </doc>
     </group>
+    <group type="NXparameters">
+        <doc>
+            Parameters used in performing the data analysis.
+        </doc>
+    </group>
+    <group type="NXdata">
+        <doc>
+            The data resulting from the operation.
+        </doc>
+    </group>
+    <attribute name="default">
```

Comment:
This is not required anymore because default has been added to NXobject, NXprocess inherits from NXobject

Reply:
Thanks for pointing that out. I'm happy to remove it.